docs/references/.pages

```diff
@@ -1,6 +1,8 @@
 nav:
   - Python Adapter Library: python-lib
   - Java Adapter Library: references/java-lib/index.html
+  - python_project
+  - java_project
```

Comment:
<img width="240" alt="image" src="https://github.com/vmware/vmware-aria-operations-integration-sdk/assets/3310870/9c5fcee5-91aa-43b2-8f5a-722519e26001">

I think we should either move the adapter library documentation in here, or just link directly to the page without the folder